### PR TITLE
[FIX] web: correct searchbar translation.

### PR DESCRIPTION
**Pull request by angv-odoo**
If you create an empty database and change the language from english, and then try to search for something in, for example, apps, you will notice that "search" will be translated, but "for:" will not. This commit fixes that issue.

ticket-3553017

Before:
image

After:
image

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by rco-odoo — COMMENTED** [review]
I am sorry, but that's awful.  Yes, it is going to be translated, but will end up with meaningless sentences in many languages.
What should be translated is the whole sentence, with placeholders for the field name and the value, i.e., `"Search %(fname)s for %(value)s"`.  Notice that the verb in that sentence is "to search for", not "to search".
For instance, in French, you will end up with "Rechercher **Produit** pour: _lait_", which does not mean anything.  A possible valid translation is "Rechercher _lait_ dans **Produit**".

**Comment by angv-odoo** [comment]
Hi @rco-odoo, first of all, thanks for the comment :) 
 Currently, in any version of runbot up to master, french "search for" is translated as "Rechercher X pour:":
image

I understand that it is not correct, in other languages translations "for" and "at" also seem to be translated literally, and it needs a fix, but I'm not sure what would be a good solution for it: phrase "search for" does not make sense in some languages like French or most Slavic languages( It should be "search %(value) in %(category)" instead of "search %(category) for %(value)" there), and in some cases with Asian languages we may need to add suffixes on search word instead (For example, crude structure in Georgian would be "search %(value) %(category)-in" ). 

The original issue that the ticket was reopened for though was a bit different: If you do not have installed hr_timesheets, loyalty, rating, stock_accounting, point_of_sale, or social installed on the Odoo database, "for" and "at" will not be translated, even though it is used in the web module. Therefore, when you search for something, let's say, in German, in the search bar you get some german-english amalgamation instead of a proper(but possibly still wrong) phrase.

TLDR: The current way of having search bar translations may be right for some languages but is wrong for most, and needs to be fixed, but it may be a bit too big of scope compared to the issue I'm trying to solve.
